libraries/Microsoft.Bot.Builder.Core.Extensions/ShowTypingMiddleware.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Core.Extensions
+{
+    public class ShowTypingMiddleware : IMiddleware
```

Comment:
>ShowTypingMiddleware [](start = 17, length = 20)

I like this one too.  Add comment describing usage?

Reply:
Description added.